modules/admin-ui/src/test/resources/karma.conf.js

```diff
@@ -52,14 +52,41 @@ module.exports = function (config) {
 
         autoWatch : true,
 
-        frameworks: ['jasmine'],
+        frameworks: ['detectBrowsers', 'jasmine'],
 
-        browsers : ['PhantomJS'],
+        detectBrowsers: {
+            enabled: true,
+            usePhantomJS: true,
+            preferHeadless: true,
+            // post processing of browsers list
+            // here you can edit the list of browsers used by karma
+            postDetection: function(availableBrowsers) {
+                /* Karma configuration with custom launchers
+                customLaunchers: {
+                    IE9: {
+                        base: 'IE',
+                        'x-ua-compatible': 'IE=EmulateIE9'
+                    }
+                }
+                */
```

Comment:
This can be removed, right?

Reply:
Sure, I left it in so that if we need to define one at some point we would have a template, but I can remove it.

Reply:
Let's remove it. I'm pretty sure we don't want a custom IE launcher at any point ;-)